cl/recap/mergers.py

```diff
@@ -108,12 +127,19 @@ def find_docket_object(
             continue  # Try a looser lookup.
         if count == 1:
             d = ds[0]
-            break  # Nailed it!
+            d = confirm_docket_number_core_lookup_match(
+                kwargs, d, docket_number
+            )
```

Comment:
I think it might make more sense if we tweak this a bit, so it's something like:

```
if docket_number_core:
    d = confirm_docket_number_core_lookup(...)
```

It feels more clear this way, because otherwise you're doing the `confirm...` work on all look-ups, which is confusing (even though they don't do anything).

Reply:
Yeah, this is clearer, I also added the pacer_case_id in this `if`:
`if kwargs["pacer_case_id"] is None and kwargs.get("docket_number_core", None):`

Since this confirmation should only occurs when the pacer_case_id is None in the lookup, since the `pacer_case_id` has priority over the `docket_number`, if for any reason a the docket_number changes in a docket if the `pacer_case_id` matched we don't need to confirm the `docket_number`.